graphene_django_extras/utils.py

```diff
@@ -214,13 +215,23 @@ def is_required(field):
 
     except AttributeError:
         return False
+    tag_field_names = [
+        'asset', 'category', 'config', 'created_by', 'description', 'last_modified_by',
+        'parent_tags', 'short_name', 'span_value', 'value_table', 'write_back_tag',
+        'zero_value'
+    ]
+    asset_op_def_fields = ['alarm_tags', 'batch_tag', 'cycle_phase_tag',
+                    'cycle_tag', 'start_state', 'stop_state']
+    if field.model.__name__ == 'AssetOperationDefinition' and field.name in asset_op_def_fields:
```

Comment:
@tushar-quartic @shivinquartic Pls explain in detail why this can't be handled & why is this hardcode required?

Reply:
I think we found a way to do so dynamically. This is not required. Right @tushar-quartic . Please close the PR if that is the case and raise a PR once the changes are done as discussed.

Reply:
Ok closing this PR